src/generate-ova-from-template.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+import struct
+import tempfile
+import os
+import sys
+import json
+from subprocess import check_call
+from stat import *
+from shutil import rmtree
+import tarfile
+
+# Usage: generate-ova-from-template.py input_image.qcow2 input_template.xml example_ova_parameters.json output_image.ova
+
+def qcow_disk_size(filename):
```

Comment:
can we replace this function with a call to `qemu-img info` ? I think I'd rather reduce the complexity even if it does mean calling an external binary:

```
$ qemu-img info /var/b/images/fedora-coreos-29.46-qemu.qcow2 --output json | jq '."virtual-size"'
8589934592

```

Reply:
I have no objection to that change if you'd like to make it prior to merging.

Reply:
+1 thanks! we'll make it in a followup PR